mantidproject/mantid
# Remove the save menu option for SaveASCII v1

**Description of work.**

Removed the save menu option for SaveASCII v1
From the workspaces toolbox in plot and workbench



**To test:**

1. for both mantidplot and workbench
1. Load some data or create a workspace
1. with the workspace selected in the workspace toolbox
1. click the save button, only Nexus and Ascii should be options
1. Ascii should launch the SaveAscii v2 dialog box.

Fixes #27619. 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
